implementations/micrometer-registry-humio/src/test/java/io/micrometer/humio/HumioMeterRegistryTest.java

```diff
@@ -16,23 +16,40 @@
 package io.micrometer.humio;
 
 import com.github.tomakehurst.wiremock.WireMockServer;
+import io.micrometer.core.instrument.FunctionCounter;
```

Comment:
I'm not expert here but looks like Intelij default auto-collapse should kick in here 
![image](https://user-images.githubusercontent.com/4628962/51513058-b38e9080-1dbd-11e9-9f62-737eaa021413.png)

Reply:
@OleksiiBondar I'm not sure there's any rule for enforcing to use `*` in this repo. I usually don't use it. If there's any rule for it, I'm happy to comply.

Reply:
Let's tackle standardizing and enforcing a rule for this separate from this PR.